### PR TITLE
feat: Support new chain on Cayenne

### PR DESCRIPTION
**Pull request by Adarsh-Kumar28**
# Description

Support Verify Testnet on Cayenne.

## Type of change

- [X] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Manual tests

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules (Pending)

1. https://github.com/LIT-Protocol/lit_node_admin_app/pull/43
2. https://github.com/LIT-Protocol/lit_node_admin_app/pull/45

### CODE REVIEW

File: typedoc.json [hunk 1]
```diff
@@ -30,4 +30,4 @@
   ],
   "out": "doc",
   "skipErrorChecking": true
-}
\ No newline at end of file
+}
```
Comment (rahul-ramesh) [inline typedoc.json:33]: trailing newline?

**Review by rahul-ramesh — APPROVED** [review]

File: typedoc.json (hunk 1 above)
Comment (Adarsh-Kumar28) [reply typedoc.json:33]: Prettier

**Review by MaximusHaximus — APPROVED** [review]
:+1: LGTM!

File: typedoc.json (hunk 1 above)
Comment (rahul-ramesh) [reply typedoc.json:33]: 👍

**Comment by Ansonhkg** [comment]
LGTM

**Pull request merged**